ci-operator/config/redhat-openshift-ecosystem/msp/redhat-openshift-ecosystem-msp-ocp-4.8.yaml

```diff
@@ -22,12 +22,13 @@ tests:
   cron: '@yearly'
   steps:
     dependency_overrides:
-      BUNDLE_IMAGE: brew.registry.redhat.io/rh-osbs-stage/e2e-e2e-test-operator-bundle-container:8.0-11
-      OO_INDEX: brew.registry.redhat.io/rh-osbs-stage/iib:40523
+      BUNDLE_IMAGE: brew.registry.redhat.io/rh-osbs/integr8ly-integreatly-operator:v1.15.0-19
```

Comment:
Are we sure that such build will not be garbage collected from brew in few weeks/months?

Reply:
the same applies to other occurrences of the same image.

Reply:
Are you proposing storing the image in the cvpops repository on quay?

Reply:
We need to avoid surprises (false positives) cause by removed image from registry.
There are more ways how to achieve it.

Released bundles will never be garbage collected from brew and thus also will be always available in brew.registry.redhat.io.

Using different registry is possibility as well.

Reply:
I reverted to the use of the standard e2e test bundle since the MSP bundles are not currently ready for testing in any case